app/partials/events.html

```diff
@@ -0,0 +1,24 @@
+<div cg-busy="{promise: loadingPromise, templateUrl:'partials/loadingtemplate.html'}">
+<span ng-if="events">
```

Comment:
This isn't valid HTML 5 — You can't have block elements like `div` or `section` inside an inline element like `span`. This should be a `div`.